src/gateways/beaconsApiGateway.ts

```diff
@@ -0,0 +1,21 @@
+import axios from "axios";
+
+export class BeaconsApiGateway {
+  private apiUrl: string;
+  private registrationsEndpoint = "registrations/register";
+
+  constructor() {
+    this.apiUrl = process.env.API_URL;
+  }
+
+  public async sendRegistration(json: any): Promise<boolean> {
+    const url = `${this.apiUrl}/${this.registrationsEndpoint}`;
+
+    try {
+      await axios.post(url, json);
+      return true;
+    } catch (error) {
+      return false;
+    }
+  }
+}
```

Comment:
What's your thinking behind the `any` type here?  Is this pending resolution of the question "How do we model the domain objects (Registration, Beacon, Use etc.)?" in the tech debt meeting today?

Reply:
Yeah exactly that.  I had created the objects within a `model/registration` file but thought that it may have been rework that we would have to do post the tech-debt meeting.

Reply:
Excellent.  Could you add a TODO to catch this when we've got a way forward?

Reply:
Added TODO